crates/astria-sequencer/README.md

```diff
@@ -21,18 +24,32 @@ CC=/usr/bin/gcc-12 CXX=/usr/bin/c++-12 cargo build
 <https://github.com/rust-rocksdb/rust-rocksdb/issues/713>
 <https://github.com/facebook/rocksdb/pull/11118>
 
-## Usage
+
+## Running the Sequencer
+
+### Configuration
+
+Composer is configured via environment variables. An example configuration can
+be seen in `local.env.example`.
+
+To copy a configuration to your `.env` file run:
+
+```sh
+
+# Can specify an environment
+just copy-env <ENVIRONMENT>
+
+# By default will copy `local.env.example`
+just copy-env
+```
 
 ### Install cometbft
 
 Ensure `~/go` is in your `PATH`, or `GOPATH` is set to some other place in your
 `PATH`.
 
 ```sh
-git clone https://github.com/astriaorg/cometbft
-cd cometbft
-export GOPATH=~/go
-make install
+just install-cometbft
```

Comment:
There is no such just command. Also, I would probably leave the steps in place. We don't want to mess with people's setup (e.g. if they have a different GOPATH, like myself).